Uplift.Service/CategorySerivce.cs

```diff
@@ -1,25 +1,87 @@
-﻿using Uplift.DataAccess.Data.Repository.IRepository;
+﻿//using Uplift.DataAccess.Data.Repository.IRepository;
+//using Uplift.Models;
+
+//namespace Uplift.Service
+//{
+//    public class CategoryService : ICategoryService
+//    {
+//        private readonly IUnitOfWork _unitOfWork;
+
+//        public CategoryService(IUnitOfWork unitOfWork) 
+//        {
+//            _unitOfWork = unitOfWork;
+//        }
+
+//        public void Create(Category model)
+//        {
+//            _unitOfWork.Category.Add(model);
+//            _unitOfWork.Save();
+//        }
+//        public IEnumerable<Category> GetAllCategories()
+//        {
+//            return _unitOfWork.Category.GetAll();
+//        }
+//    }
+//}
+
+using Microsoft.EntityFrameworkCore;
+using Uplift.DataAccess.Data;
 using Uplift.Models;
 
 namespace Uplift.Service
 {
     public class CategoryService : ICategoryService
     {
-        private readonly IUnitOfWork _unitOfWork;
+        private readonly ApplicationDbContext _context;
```

Comment:
In the service layer, we don't use the context we use unitOfWork instead. It doesn't mean that it is wrong but based on our architecture we use UOW repository pattern.

Reply:
Thank you for note. I wrote it  down.